Tests/Data/ArraysTest.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+namespace Framework\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Zest\Data\Arrays;
+
+class ArraysTest extends TestCase
+{
+    public function testIsReallyArray()
+    {
+        $this->assertTrue(Arrays::isReallyArray([0, 1]));
+        $this->assertTrue(Arrays::isReallyArray(['name' => 
+            'someName', 'username' => '@username']));
```

Comment:
This array should be constructed.

It looks like this don't have the proper coding style.

Reply:
What do you means, could you explain more please?

Reply:
It looks like the `'name' => 'someName'` is not at the same line.

Could you fix this?